### ISSUE TITLE
Auto Play (New feature request)

### ISSUE THREAD

**Issue by dreh** [senshu/Sozi#115]
I got really excited when i found out  Sozi and started to play with it, and as a graphic designer, that got me thinking about going further with it. Would be great to be able to auto play the slides.

Besides the navigation as it is today,  we could have a button to start the auto play  (like pressing "A"  on the keyboard), and it would go through the slides automatically.

We could also set a checkbox in inkscape to start the auto play as soon as the file is completely loaded in the browser, working like an animation. Also if you could set a loop checkbox, it would work as a web banner. (replacing animated gifs and flash).

So to sum up:
1 - auto play triggered by keyboard button;
2 - auto play start after file completely loaded in the browser;
3 - auto loop through the svg file

Keep up the great work.! And I hope it is easy to implement this idea :)
